Tribler/community/multichain/statistics/statistics_database.py

```diff
@@ -0,0 +1,62 @@
+"""
+This file contains the functions to get the trust statistics from the database implemented in Tribler.
+"""
+from Tribler.community.multichain.database import MultiChainDB
+
+
+class StatisticsDB(MultiChainDB):
```

Comment:
Maybe it is more efficient to replace the existing database at once rather than inheriting from the existing database first and then removing it.